tests/utils.ts

```diff
@@ -120,6 +129,18 @@ export const makeSetResultMessage = (
 	return message;
 };
 
+export const makeRefundMessage = (
+	marketAddress: string,
+	betIndex: BigNumberish
+): string => {
+	const b16Refund = formatBytes16String("refund");
```

Comment:
as i said earlier this seems unnecessary to me

Reply:
See above